src/templates/filtersDrawer.jsx

```diff
@@ -0,0 +1,674 @@
+
+                                <div
+                                    className="nested_toggles_wing--options"
+                                >
+                                    {
+                                        !isNestedTogglesOptionsEmpty &&
+                                        _.map(nestedTogglesData.options, (option) => {
+                                            nestedTogglesOptionLabelKeyNum += 1;
+                                            const isSelected = _.some(
+                                                nestedTogglesData.value,
+                                                option,
+                                            );
+
+                                            return (
+                                                <FiltersDrawerNestedTogglesWingOptionLabel
+                                                    key={nestedTogglesOptionLabelKeyNum}
+                                                    isSelected={isSelected}
+                                                    onClick={
+                                                        this.onNestedTogglesWingOptionLabelClick
+                                                    }
+                                                    option={option}
+                                                />
+                                            );
+                                        })
+                                    }
+                                </div>
+                            </Drawer.Content>
+                        </Drawer.Wing>
+                    )}
+                >
+                    <Drawer.TitleBar
+                        className="page_filters_drawer--title_bar"
+                        closeButton={(
+                            <Icon
+                                className="page_filters_drawer--close_button"
+                                compact
+                                onClick={onClose}
+                                type="times"
+                            />
+                    )}
+                        title={(
+                            <div className="page_filters_drawer--actions" title="Filters">
+                                <div
+                                    className="page_filters_drawer--clear_column"
+                                >
+                                    {/* eslint-disable max-len */}
+                                    {/* eslint-disable jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions, jsx-a11y/anchor-is-valid */}
+                                    {canClear ? (
+                                        <a
+                                            className={clearFiltersClasses}
+                                            onClick={this.onClearClick}
+                                        >
+                                            Clear Filters
+                                        </a>
```

Comment:
I'm wondering if we need to be able to start styling buttons to look like links instead of using `<a>` tags and always having to suppress all those accessibility warnings.  Clearly not the hottest priority, but it does seem like we're "doing it wrong", perhaps.

Reply:
I don't believe we're "doing it wrong" necessarily, but I do agree we need to figure out a different approach. I was actually thinking about the `<Link />` react-router component, https://knowbody.github.io/react-router-docs/api/Link.html. We'd need to look more into the component.

Reply:
Wouldn't that just be masking the problem? :-)

Reply:
I guess I mean "doing it wrong" from an accessibility standpoint: in other words, are we using hyperlinks in a way that's not intended, and this has accessibility implications.  That's been my understanding of the warnings.  Maybe I'm reading too much into it.  We've not made accessibility a priority at all, so suppressing the warnings is the right approach in the short-term ... I guess I'm wondering what we would do if we were to actually heed the warnings?